httpfuzzerprocessor/pitchWork.py

```diff
@@ -0,0 +1,97 @@
+# To resolve problem at from https://github.com/zaproxy/zaproxy/issues/2967
+# The script fuzz in mode pitchfork.
+# To Use : Enable script.
+# In fuzzer Add number multiple EmptyNull payloads with a good number of iterations.
+# Select the desired number of payloads [limit 2 to 20]
+# Select the desired number of files    [limit 2 to 20]
+
+from java.nio.file import Paths
+from javax.swing import JFileChooser
+from javax.swing import JOptionPane
+from org.zaproxy.zap.extension.fuzz.payloads.generator import FileStringPayloadGenerator
+
+payloads1 = None
+payloads2 = None
+init = False
+
+def processMessage(utils, message):
+    global number, payloads, init
+    
+    if not init:
+        initialise()
+    
+    # Stop if has end of payloads
+    for i in range(number):
+        # if end of payload stop fuzzing
+        if not payloads[i].hasNext():
+            utils.stopFuzzer()
+            # close all payload
+            for j in range(number):
+                payloads.close()
+            return
+    
+    for i in range(number):
+        # Get the next value of payloads
+        # Get information of body and replace with payload value
+        payloadNext = payloads[i].next().getValue()
+        body = message.getRequestBody().toString()
+        body = body.replace(utils.getPaylaods().get(i).getValue(), payloadNext)
+        # Set payload value to show in Fuzzer 
+        utils.getPaylaods().set(i,payloadNext)
+        # set payload in body
+        message.getRequestBody().setBody(body)
+        message.getRequestHeader().setContentLength(message.getRequestBody().length())
+
+def processResult(utils, fuzzResult):
+    return True
+
+def chooseFile():
+    fileChooser = JFileChooser()
+    fileChooser.setMultiSelectionEnabled(True)
+    filePath = ""
+    result = fileChooser.showOpenDialog(None)
+
+    if result == JFileChooser.APPROVE_OPTION:
+        selectedFiles = fileChooser.getSelectedFiles()
+        for file in selectedFiles:
+            filePath = file.getAbsolutePath()
+            print('The path is :', filePath)
+
+    return filePath
+
+def chooseNumber():
+    number = JOptionPane.showInputDialog(None, "How many payload do you want [2 to 20]:", "Input", JOptionPane.QUESTION_MESSAGE)
+    
+    # Check number between 2 to 20
+    if int(number) > 1 and int(number) < 21:
+        number = int(number)
+        return number
+    else:
+        JOptionPane.showMessageDialog(None, "Choose number between 2 to 20")
+        chooseNumber()
+
+def initialise():
+    global init
+    global payloads
+    global number
+    
+    payloads = []
+    filePaths = []
+
+    # input number of payloads
+    number = -1
+    while number == -1:
+        number = chooseNumber()
+
+    # choose file user        
```

Comment:
user file?

Reply:
Yes, is here the files of users are add.   i have modified the comment by "add files chosen by the user"